executor/analyze.go

```diff
@@ -86,6 +89,24 @@ const (
 	maxSketchSize       = 10000
 )
 
+var (
+	memToGC = atomicutil.NewInt64(0)
+	inGC    = atomicutil.NewBool(false)
+)
+
+func tryGC(memTracker *memory.Tracker, force bool) {
```

Comment:
Can you provide the comparing results between manual gc or not?

Reply:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/5151781/167099820-69787dcf-b322-497e-b2b9-b8cff528390c.png">

And if we have half memory used by long-living cache, GOGC=100 is dangrous.

Reply:
And if I just do tracker.Consume(-xxxDataSize) at the end of func instead of recordMemToGC and call runtime.GC later, it will be:
<img width="647" alt="image" src="https://user-images.githubusercontent.com/5151781/167107151-20f7b026-8961-4af8-bca4-4003f8c5c14e.png">
Since the latency of GC, heapInUse is much larger than analyze tracked.